internal/resources/smbshare.go

```diff
@@ -191,7 +191,8 @@ func (m *SmbShareManager) Finalize(
 	ctx context.Context,
 	instance *sambaoperatorv1alpha1.SmbShare) Result {
 	// ---
-	cm, err := getConfigMap(ctx, m.client, m.cfg.WorkingNamespace)
+	destNamespace := instance.Namespace
```

Comment:
IMO, it would be easier to read if you used instance.Namespace as an argument in the next line.

Reply:
It's there to help me remember things better. I'll think I will leave it for now, thanks.